Formula/liblwgeom.rb

```diff
@@ -1,20 +1,10 @@
 class Liblwgeom < Formula
   desc "Allows SpatiaLite to support ST_MakeValid() like PostGIS"
   homepage "https://postgis.net"
-  revision 3
 
   stable do
```

Comment:
`stable do ... end` is no longer needed

Reply:
Fixed in e5f861fee08d266f9cb39ddd2e853443217e3ebd